src/mysql.erl

```diff
@@ -175,32 +181,86 @@ start_link(Options) ->
          Query :: iodata(),
          Result :: query_result().
 query(Conn, Query) ->
-    query_call(Conn, {query, Query}).
+    query_call(Conn, {query, Query, undefined, undefined}).
```

Comment:
Suggesting more descriptive atom names, e.g. `no_filtermap_fun` and `default_timeout`. (See also other comments about this.)

Reply:
I have no preference here, can do :) See my comment on (not) allowing `undefined`, though.